src/device-state.ts

```diff
@@ -41,6 +41,8 @@ import {
 import * as dbFormat from './device-state/db-format';
 import * as apiKeys from './lib/api-keys';
 
+const disallowedPatchFields = ['local_ip', 'local_port'];
```

Comment:
Could you move this to host-config.ts? It doesn't make much sense here.

Reply:
This array is being used locally in this module, in `router.patch`. I've modified it to be this way since we decided to remove it from constants.ts.

Reply:
Then maybe we should call it something `disallowedHostConfigPatchFields`?  Since this is (for now) in the device state module it might be better to be more verbose.

Reply:
Only this is missing and we can get this merged :)

Reply:
Done!